### PR TITLE
Added Homepage

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**
Quick HomePage review!

https://user-images.githubusercontent.com/53683977/154133529-3752cf5a-82d2-4567-916d-8e06efe0eb70.mp4

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+    max-height: 100vh;
+    overflow-y: scroll;
+    border: 1px solid gray;
+    scroll-snap-type: y mandatory;
+  }
+  .scroll-item {
+    height: 100vh;
+    scroll-snap-align: start;
+    color: rgb(87, 28, 28);
+    text-align: center;
```
Comment (ark2002) [inline style.css:30]: try following a single color refrences dont mix rgb and hex

File: index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                        <div class="menu-item">Anime</div>
+                        <div class="menu-item">English-Series</div>
+                        <div class="menu-item">Bollywood</div>
+                        <div class="menu-item">Hollywood</div>
+                    </div>
+                    <div class="category-item-cards">
+                        <div class="card-item">
+                            <div class="img-cont">
+                                <img src="https://www.imagedecor.com.au/wp-content/uploads/2018/04/PP32769_f.jpg" alt="" srcset="">
+                            </div>
```
Comment (ark2002) [inline index.html:52]: it's recommended to use alt attribute to help the  user incase the image does'nt load up

**Review by ark2002 — CHANGES_REQUESTED** [review]
some changes recommended

File: style.css (hunk 1 above)
Comment (HrishikeshMahalle) [reply style.css:30]: Since these changes are in main styles i'll upadge them once merged. Thank you for PR

File: index.html (hunk 2 above)
Comment (HrishikeshMahalle) [reply index.html:52]: Yes sure

**Review by md-amir-gauhar — APPROVED** [review]
Looks great. Try to use semantic tags.

**Review by muskan-gupta-0803 — APPROVED** [review]
Looks good to merge.

Just a suggestion to add alt text to images and you can also remove unnecessary spaces/blank lines from styles.css

**Pull request merged**